atlp-rwanda/tech-strikers-bn-backend
# #175974127 It sets up github Actions as CI Tool

#### What does this PR do?
It integrates our Repo with Github Actions CI for Node js
#### Description of Task to be completed?
- created `.github/workflows/nodejs.yml` file for CI configurations
- I added environment variables in the above file
- Added Github Actions Status Badge
#### How should this be manually tested?
- run : 
  `git clone https://github.com/Duncanwar/blog-backend.git`
 `npm install`
`git checkout ch-github-actions-setup-175974127`
`npm run dev`
`git push`
You come to this repo remote branch and you will see a badge showing that the tests are passing successfully.
#### Any background context you want to provide?
Coveralls is not working
#### What are the relevant pivotal tracker stories?
[#175974127](https://www.pivotaltracker.com/story/show/175974127)
#### Screenshots (if appropriate)
![Screenshot from 2020-12-02 19-53-20](https://user-images.githubusercontent.com/57761095/100911551-1122b680-34d8-11eb-8bc7-236e454da1fa.png)
![Screenshot from 2020-12-02 19-53-06](https://user-images.githubusercontent.com/57761095/100911562-12ec7a00-34d8-11eb-9b0a-33c7383b128e.png)
![Screenshot from 2020-12-02 19-52-55](https://user-images.githubusercontent.com/57761095/100911569-141da700-34d8-11eb-83ed-a735ccb1339b.png)

#### Questions:
